src/azure-cli/azure/cli/command_modules/network/private_link_resource_and_endpoint_connections/custom.py

```diff
@@ -31,9 +31,10 @@ def register_providers():
     _register_one_provider('Microsoft.EventGrid/domains', '2020-04-01-preview', True)
     _register_one_provider('Microsoft.SignalRService/signalr', '2020-05-01', False)
     _register_one_provider('Microsoft.Network/applicationGateways', '2020-05-01', True)
+    _register_one_provider('Microsoft.Compute/diskAccesses', '2020-05-01', False, support_manually_approve=False)
```

Comment:
should it be `support_manual_operation` since it's not only related with approve.

Reply:
good suggestion, will adjsut it

Reply:
How about we call it `support_connection_operation`? Which one is better?

Reply:
good choice~